### PR TITLE
Static texture classes are now proper CustomManagers

### PR DESCRIPTION
**Pull request by kvakvs**
* Texture classes are not static anymore
* On level loaded, all textures are loaded too
* On level unloaded, textures are freed

In debug build this creates visible delay. Should we show a progress indicator or some short-time notification to let the player know that we're loading?

### CODE REVIEW

File: TLM/TLM/UI/Textures/RoadSignThemes.cs [hunk 1]
```diff
@@ -285,14 +297,13 @@ public Texture2D Parking(bool p) => this.parking_.ContainsKey(p)
         }
 
         // TODO: Split loading here into dynamic sections, static enforces everything to stay in this ctor
```
Comment (aubergine10) [inline TLM/TLM/UI/Textures/RoadSignThemes.cs:299]: Is this comment still relevant?

Comment (kvakvs) [reply TLM/TLM/UI/Textures/RoadSignThemes.cs:299]: No its not static anymore

**Review by krzychu124 — APPROVED** [review]
I didn't notice any issue with regards to memory leaking, or hot-reloading. Everything works flawlessly.

**Review by chameleon-tbn — APPROVED** [review]
all seems to work flawless with this build, therefor i give it a go.

**Pull request merged**